clients/src/main/java/org/apache/kafka/common/requests/OffsetFetchResponse.java

```diff
@@ -49,9 +49,9 @@
      *
      *  UNKNOWN_TOPIC_OR_PARTITION (3)  <- only for request v0
      *  OFFSET_LOAD_IN_PROGRESS (14)
-     *  NOT_COORDINATOR_FOR_CONSUMER (16)
```

Comment:
I think we would better return GROUP_COORDINATOR_NOT_AVAILABLE for (!isActive), and NOT_COORDINATOR_FOR_GROUP for (!isCoordinatorForGroup(groupId)) for all requests / conditions, but on the server side we should check the latter first before the former, so that consumers will get NOT_COORDINATOR_FOR_GROUP if it is ever the case and rediscover.